frontend/src/components/MapView/DateSelector/index.tsx

```diff
@@ -340,8 +340,7 @@ const styles = (theme: Theme) =>
     container: {
       position: 'absolute',
       bottom: '8%',
-      zIndex: theme.zIndex.modal,
-      width: '100vw',
+      width: '100%',
```

Comment:
Can you explain this change? There might be a reason why we were using 100vw initially